docs/source/tutorials/text.ipynb

```diff
@@ -56,7 +106,7 @@
    "source": [
     "# Package installation (hidden on docs website).\n",
     "# If running on Colab, may want to use GPU (select: Runtime > Change runtime type > Hardware accelerator > GPU)\n",
-    "\n",
+    "# Package versions we used: tensorflow==2.8.1 keras==2.8.0 scikeras==0.6.1\n",
```

Comment:
don't think keras needs to be declared anymore nowadays, it's just part of tensorflow

Reply:
I've removed it